documentation/tutorials/api-keys.md

```diff
@@ -0,0 +1,122 @@
+# API Keys
+
+## A note on API Keys
+
+API keys are a bit different from the other strategies in AshAuthentication, in that it is your job to create and manage them.
+
+This means that to leverage them in the standard way, you will need to build a UI for managing API keys. You can create API keys in iex in development or if you don't have a need for users to create their own API keys.
+
+It is up to you to determine what constitutes a valid API key, and who is allowed to create/modify them. All that we require is that an API key is a string attribute on the related api key resource. Be sure that the API key relationship used with this strategy only includes valid API keys.
+
+For example:
+
+```elixir
+has_many :valid_api_keys, MyApp.Accounts.ApiKey do
+  filter expr(is_expired == false)
+end
+```
+
+## Installation
+
+<!-- tabs-open -->
+
+### With Igniter (recommended)
+
+Use `mix ash_authentication.add_strategy api_key` to install this strategy, and modify the generated resource
+to suit your needs.
+
+### Manually
+
+#### Create an API key resource
+
+```elixir
+defmodule MyApp.Accounts.ApiKey do
+  use Ash.Resource,
+    data_layer: AshPostgres.DataLayer,
+    authorizers: [Ash.Policy.Authorizer]
+
+  actions do
+    defaults [:read, :destroy]
+
+    create :create do
+      primary? true
+      accept [:api_key, :user_id, :expires_at]
+    end
```

Comment:
I would probably deter users from being able to chose their own tokens. Could we remove `api_key` from `accept` by default and provide a change that chooses a random token with an ok amount of entropy?